docs/DeveloperGuide.md

```diff
@@ -166,6 +166,26 @@ The following sequence diagram shows how the add event operation works.
 
 This operation is similar to that of adding a person. Adding an event involves calling `Model#addEvent(Event)`, which in turn calls `AddressBook#addEvent(Event)` to add the Event object to the existing `AddressBook`.
 
+### \[Implemented] Sort events feature
+
+#### Current Implementation
+
+Sorting a list of events is a feature that uses the command `sortevent a/b/c/d`.
```

Comment:
I think some may mistaken this for actually adding 'sortevent a/b/c/d' since most of our existing command uses '/' in the prefix. Alternatively, you could list them out explicitly, 'sortevent a', 'sortevent b', ... and so on

Reply:
Thanks! I have updated 😃